templates/ovncontroller/bin/stop-vswitchd.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+#
+# Copyright 2024 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+set -x
+
+ovs_folder=/var/lib/openvswitch
+
+if [ -f $ovs_folder/flows ]; then
+    rm -rf $ovs_folder/flows
+fi
+if [ -f $ovs_folder/bridges ]; then
+    rm -rf $ovs_folder/bridges
+fi
+if [ -d $ovs_folder/saved_flows ]; then
+    rm -rf $ovs_folder/saved_flows
+fi
+
+ovs-vsctl -- --real list-br > $ovs_folder/bridges
```

Comment:
Do we ever create "fake" bridges? Why do we need to explicitly filter them out with `--real` argument?

Reply:
I did it this way to keep it as similar as possible to the original reload in ovs: https://github.com/openvswitch/ovs/commit/ea36b04688f37cf45b7c2304ce31f0d29f212d54#diff-dcb4491325151a3044b5073d4419d9ffe2a9af86c6ba6e042772d7b32cd303e9
We can remove it if unneeded

Reply:
OK we'll leave it, I will only add a note that this may be redundant.